tailwind.config.js

```diff
@@ -1,7 +1,17 @@
+const colors = require('tailwindcss/colors');
+
```

Comment:
Does using prefilled values makes sense or should it be initialized with empty values just as stated here: https://tailwindcss.com/docs/guides/gatsby

Reply:
I've used values as defined by @harsh242 and Ayush in the Figma files.

Reply:
Got it.